.travis.yml

```diff
@@ -6,52 +6,57 @@ language: rust
 services: docker
 sudo: required
 
-# This is the Rust channel that build jobs will use by default but can be
-# overridden on a case by case basis down below
-rust: 1.13.0
```

Comment:
If you use the default, you end up with an "extra" test run. You can see how there's two `x86-64-unknown-linux-gnu` runs that execute right now. This fixes that problem. Looks like I didn't specify that in the commit message, which I can do.